### PR TITLE
comments added

### PR DESCRIPTION
**Pull request by kritika81**

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CommentsController < ApplicationController
+def index
+end
+def new
+@comment=Comment.new
```
Comment (OnShuffle) [inline app/controllers/comments_controller.rb]: indentation
spcaing between "="

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+end
+def create
+  @comment = Comment.new(params[:comment])
+ 
+  @comment.save
+  redirect_to @comment
```
Comment (OnShuffle) [inline app/controllers/comments_controller.rb]: redirect_to string parameter leta h, jahan redirect krna h
aur wo string rails helper se bna
example root_path ek rails helper h. (helper matlab helping function)
`puts root_path` ========>  returns "/"
`redirect_to root_path`
will redirect to "/" matlab localhost:3000/
ise aise bhi likh sakte h
`redirect_to "/"`
same cheez h
ab tujhe jahan redirect krna h wahan ka path (relative path, localhost nahi) de.

File: config/routes.rb [hunk 3]
```diff
@@ -5,6 +5,8 @@
 get 'welcome/sleepandfood', to: 'welcome#sleepandfood'
 get 'welcome/relationship',to: 'welcome#relationship'
 get 'welcome/parenting', to: 'welcome#parenting'
+resources :comments
+get 'welcome/comments', to: 'comments#new'
```
Comment (OnShuffle) [inline config/routes.rb]: iski jaroorat kyu h?
`resources :comments`   will automatically create path "/comments/new"

File: db/migrate/20170712100444_create_comments.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[5.1]
+  def change
+    create_table :comments do |t|
+      t.string :commenter
```
Comment (OnShuffle) [inline db/migrate/20170712100444_create_comments.rb]: what is commenter

File: db/migrate/20170712100444_create_comments.rb [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+      t.string :body
```
Comment (OnShuffle) [inline db/migrate/20170712100444_create_comments.rb]: yahan body likha h upper form mein title likha h.... error aayegi, same likhna hota h tabhi pta chalta h use

File: app/views/comments/new.html.erb [hunk 6]
```diff
@@ -0,0 +1,12 @@
+
+<h1>new comment</h1>
+<%= form_for :comments, url: comments_path do |f| %>
+ <p>
+    <%= f.label :writecomment %><br>
+    <%= f.text_field :title %>
```
Comment (OnShuffle) [inline app/views/comments/new.html.erb]: body or title?

File: db/migrate/20170712100444_create_comments.rb (hunk 4 above)
Comment (OnShuffle) [reply db/migrate/20170712100444_create_comments.rb]: sep model bna user

File: config/routes.rb (hunk 3 above)
Comment (kritika81) [reply config/routes.rb]: arey localhost:3000/comments/new 
aisa path dene mai mera output aa ja rha h...par mjhe phle welcome page kholna h..phir health mai click krna h to nya page khulega..us page mai comment likhne hain

File: db/migrate/20170712100444_create_comments.rb (hunk 4 above)
Comment (kritika81) [reply db/migrate/20170712100444_create_comments.rb]: kya mtlb?

File: app/controllers/comments_controller.rb (hunk 2 above)
Comment (kritika81) [reply app/controllers/comments_controller.rb]: abe 
Unknown action
The action 'show' could not be found for CommentsController
ye error a ri

mai chah ri hu ki jb comments save ho jaye to phir se new comment wali html display ho...tb kra h ...redirect_to @comments

Comment (OnShuffle) [reply app/controllers/comments_controller.rb]: pehele to nomenclature seekh
comment wali html kya hota h?
"new comment link mein redirect krna h" aisa bol

Comment (OnShuffle) [reply app/controllers/comments_controller.rb]: ab bta new comment wala link kya h?
example: /new/comments        ?

agar ye h to `redirect_to "/new/comments"`

File: db/migrate/20170712100444_create_comments.rb (hunk 4 above)
Comment (OnShuffle) [reply db/migrate/20170712100444_create_comments.rb]: seperate model bna user
user will be commenter
matlab a comment belongs to user and user can has many comments

File: app/models/admin.rb [hunk 7]
```diff
@@ -0,0 +1,6 @@
+class Admin < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
```
Comment (OnShuffle) [inline app/models/admin.rb:5]: validatable ki jaroorat nahi h
isko daal ke ye ek mail bhejeka registration ke baad email validate krne ke liye
abhi hata de

File: config/routes.rb [hunk 8]
```diff
@@ -1,12 +1,14 @@
 Rails.application.routes.draw do
+  
+  devise_for :admins
```
Comment (OnShuffle) [inline config/routes.rb:3]: admins kyu?

Comment (OnShuffle) [reply config/routes.rb:3]: user naam kyu nahi diya model ka?

**Comment by OnShuffle** [comment]
looks good

**Pull request closed without merging**